concepts/Common/Distribution Channels/05 CDN/10 Subresource Integrity Hashes.md

```diff
@@ -1,4 +1,4 @@
-You can use <a href="https://developer.mozilla.org/en-US/docs/Web/Security/Subresource_Integrity" target="_blank">Subresource Integrity</a> (SRI) when fetching DevExtreme files from CDN. Below are the files' SRI hashes for DevExtreme minor releases. Each file has a `sha384` and `sha512` hash.
+You can use <a href="https://developer.mozilla.org/en-US/docs/Web/Security/Subresource_Integrity" target="_blank">Subresource Integrity</a> (SRI) hashes to verify DevExtreme files when fetching them from CDN. Each link below contains `sha384` and `sha512` hashes:
 
 ---
 ##### 20.1.0
```

Comment:
Why we have this part of the article? We haven't published any files of v20.1 to CDN yet... @viterobk